index.html

```diff
@@ -204,24 +219,144 @@ <h3 class="project1_title">Multi-Post Stories</h3>
         </div>
 
         
-    </section> 
+    </section>
 
-    <section id="contact_info">
-        <div>
-            <h2 id="greeting_2">About Myself</h2>
+    <section class="desktop_projects">
+        <div class="desktop_project_1">
+            <div class ="desktop_project_1_img">
+                <img src="img/snapshot.svg" alt="snapshot">
+            </div>
+            <div class="desktop_project_1_content">
+
+            <h3 class="project1_title">Tonic</h3>
+
+        <ul class="tonic_list">
+            <li class="canopy">CANOPY</li>
+            <li> <img class="counter" src="img/Counter.png" alt="counter"> </li>
+            <li> Back End Dev </li>
+            <li> <img class="counter" src="img/Counter.png" alt="counter"> </li>
+            <li> 2015 </li>
+        </ul>
+        <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+
+        <div class="small_buttons">
+            <ul class="button_list">
+                <li class="language_descriptor">html</li>
+                <li class="language_descriptor">css</li>
+                <li class="language_descriptor">javaScript</li>
+            </ul>
         </div>
 
-        <div>
-            <p class="about_me">I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+        <button class="see_project" type="button">See Project</button>
+            </div>
+
+        </div>
```

Comment:
- [ ] The same remarque about the desktop version, please kindly fix it for all the `project section` section :grin: :grin:. You need to center your card and arrange the button.
 
 Your design

![Screenshot from 2022-07-06 13-51-23](https://user-images.githubusercontent.com/74501523/177546106-70542b8e-1592-422b-82ea-382444c3b66d.png)

Figma design

![Screenshot from 2022-07-06 13-51-00](https://user-images.githubusercontent.com/74501523/177546068-0ec2ad47-be55-4a7e-a266-1321fe2d1152.png)